kerl

```diff
@@ -224,115 +224,66 @@ KERL_COLORIZE=${KERL_COLORIZE:=$KERL_COLOR_AVAILABLE}
 KERL_INCLUDE_RELEASE_CANDIDATES=${KERL_INCLUDE_RELEASE_CANDIDATES:=no}
 KERL_CHECK_BUILD_PACKAGES=${KERL_CHECK_BUILD_PACKAGES:="yes"}
 
-if [ -n "$OTP_GITHUB_URL" ]; then
-    _OGU="$OTP_GITHUB_URL"
-fi
-if [ -n "$KERL_CONFIGURE_OPTIONS" ]; then
-    _KCO="$KERL_CONFIGURE_OPTIONS"
-fi
-if [ -n "$KERL_CONFIGURE_APPLICATIONS" ]; then
-    _KCA="$KERL_CONFIGURE_APPLICATIONS"
-fi
-if [ -n "$KERL_CONFIGURE_DISABLE_APPLICATIONS" ]; then
-    _KCDA="$KERL_CONFIGURE_DISABLE_APPLICATIONS"
-fi
-if [ -n "$KERL_SASL_STARTUP" ]; then
-    _KSS="$KERL_SASL_STARTUP"
-fi
-if [ -n "$KERL_DEPLOY_SSH_OPTIONS" ]; then
-    _KDSSH="$KERL_DEPLOY_SSH_OPTIONS"
-fi
-if [ -n "$KERL_DEPLOY_RSYNC_OPTIONS" ]; then
-    _KDRSYNC="$KERL_DEPLOY_RSYNC_OPTIONS"
-fi
-if [ -n "$KERL_INSTALL_MANPAGES" ]; then
-    _KIM="$KERL_INSTALL_MANPAGES"
-fi
-if [ -n "$KERL_INSTALL_HTMLDOCS" ]; then
-    _KIHD="$KERL_INSTALL_HTMLDOCS"
-fi
-if [ -n "$KERL_BUILD_PLT" ]; then
-    _KBPLT="$KERL_BUILD_PLT"
-fi
-if [ -n "$KERL_BUILD_DOCS" ]; then
-    _KBD="$KERL_BUILD_DOCS"
-fi
-if [ -n "$KERL_DOC_TARGETS" ]; then
-    _KDT="$KERL_DOC_TARGETS"
-fi
-if [ -n "$KERL_BUILD_BACKEND" ]; then
-    _KBB="$KERL_BUILD_BACKEND"
-fi
-if [ -n "$KERL_RELEASE_TARGET" ]; then
-    _KRT="$KERL_RELEASE_TARGET"
-fi
-
-OTP_GITHUB_URL=https://github.com/erlang/otp
-KERL_CONFIGURE_OPTIONS=
-KERL_CONFIGURE_APPLICATIONS=
-KERL_CONFIGURE_DISABLE_APPLICATIONS=
-KERL_SASL_STARTUP=
-KERL_DEPLOY_SSH_OPTIONS=
-KERL_DEPLOY_RSYNC_OPTIONS=
-KERL_INSTALL_MANPAGES=
-KERL_INSTALL_HTMLDOCS=
-KERL_BUILD_PLT=
-KERL_BUILD_DOCS=
-KERL_DOC_TARGETS=chunks
-KERL_BUILD_BACKEND=
-KERL_RELEASE_TARGET=
-
 # ensure the base dir exists
 mkdir -p "$KERL_BASE_DIR" || exit 1
 
+# List is [<config_key>=<def_value>, ...]
+_KERL_CFGS="\
+OTP_GITHUB_URL=https://github.com/erlang/otp \
+KERL_CONFIGURE_OPTIONS= \
+KERL_CONFIGURE_APPLICATIONS= \
+KERL_CONFIGURE_DISABLE_APPLICATIONS= \
+KERL_SASL_STARTUP= \
+KERL_DEPLOY_SSH_OPTIONS= \
+KERL_DEPLOY_RSYNC_OPTIONS= \
+KERL_INSTALL_MANPAGES= \
+KERL_INSTALL_HTMLDOCS= \
+KERL_BUILD_PLT= \
+KERL_BUILD_DOCS= \
+KERL_DOC_TARGETS=chunks \
+KERL_BUILD_BACKEND= \
+KERL_RELEASE_TARGET= \
+"
+
+save_default_kerl_cfgs() {
```

Comment:
I'm looking at this function now, and thinking I could get away with a simple `eval $_KERL_CFGS`.